Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsViewModel.kt

```diff
@@ -55,6 +56,9 @@ class PersonDetailsViewModel @AssistedInject constructor(
 
     val events = SingleLiveEvent<PersonDetailsEvents>()
 
+    private val colorShadeData = MutableLiveData(colorShade)
```

Comment:
I'd prefer a MutableStateFlow, but that is just personal preference.. (OPTIONAL)

Reply:
let us keep it LiveData from me 😃